### DISCUSSION TITLE
Card classes seem to be everywhere...How to remove from widgets?

### DISCUSSION THREAD

**Discussion by currentcreative** [bootscore/bootscore#731] [General]
The first thing I noticed when I activated the child theme is I'm going to need to remove all these borders on everything. I saw 'card' classes were used all over the index.php template, so I removed them, but that still left all the borders around the sidebar items.

I then looked in sidebar.php and inc/widgets.php and removed all the card classes I could find, but the widgets in the sidebar still have borders around them.

Can someone please explain how to remove all these borders?

**Comment by crftwrk** [comment]
Hello,

horizontal cards are used in loop, `index.php`, `archive.php` and `/template-parts/content-search.php`. You can override those templates in your child.

Sidebar widgets uses vertical cards in `/inc/widgets.php`. `function bootscore_widgets_init()` is a pluggable function which you can override in child's `functions.php`:

```php
// Sidebar
register_sidebar(array(
  'name'          => esc_html__('Sidebar', 'bootscore'),
  'id'            => 'sidebar-1',
  'description'   => esc_html__('Add widgets here.', 'bootscore'),
  'before_widget' => '<section id="%1$s" class="widget %2$s card card-body mb-4">',
  'after_widget'  => '</section>',
  'before_title'  => '<h2 class="widget-title card-header h5">',
  'after_title'   => '</h2>',
));
```

No more cards used in theme (except the WooCommerce loop).

If you just want to remove the borders from the cards, why not edit the `card` component and add `$card-border-width: 0;` to `_bscore_variable.scss`?


P.S. a link to your project is always very useful to understand what you mean and easy debugging.
